rho/factdecryptcommand.py

```diff
@@ -42,6 +42,11 @@ def __init__(self):
         self.parser.add_option("--facts", dest="facts", metavar="FACTS",
                                action="callback", callback=multi_arg,
                                default=[], help=SUPPRESS_HELP)
+
+        self.parser.add_option("--decrypted-path", dest="decrypted_path",
+                               metavar="DECRYPTEDPATH",
+                               help=_("Location for the decrypted file"))
```

Comment:
Should use consistent working this is different than encryption help:
`Destination for the encrypted file`

Reply:
I think all the new option could be named the same ``--path`` or ``--destinationfile``. The subcommand would already provide the context if the user is `encrypt`, `decrypt` or `redact`.